packages/react-server/src/ReactFlightServer.js

```diff
@@ -1267,15 +1270,27 @@ function renderModelDestructive(
           }
         }
 
+        const props = element.props;
+        let ref;
+        if (enableRefAsProp) {
+          // TODO: This is a temporary, intermediate step. Once the feature
+          // flag is removed, we should get the ref off the props object right
+          // before using it.
+          const refProp = props.ref;
+          ref = refProp !== undefined ? refProp : null;
```

Comment:
Passing it along here mostly exists for parity with Fizz in terms of file structure. It'll just error lower. We can probably just pass `null` here and disable the error.